Gemfile.tt

```diff
@@ -6,13 +6,17 @@ ruby '<%= RUBY_VERSION %>'
 # Core
 gem 'bootsnap', ">= 1.7.5", require: false
 gem 'bugsnag'
-gem 'pg'<%= gemfile_requirement("pg") %>
-gem 'puma'<%= gemfile_requirement("puma") %>
+gem 'pg', '~> 1.3.4'
+gem 'puma', '~> 5.6'
 gem 'rack-cors'
 gem 'rails', "~> <%= Rails.version %>"
 gem 'sidekiq', ">= 6.2.1"
-<% if gemfile_requirement("webpacker") -%>
-  gem 'webpacker'<%= gemfile_requirement("webpacker") %>
+<% if !@options['api'] %>
+gem 'webpacker'
```

Comment:
`webpacker` is no longer a default in Rails 7 unless explicit addition through CLI arguments when generating a new rails app. So the previous implementation with the usage of `gemfile_requirement` is correct since it only adds webpacker to the Gemfile if present in the original Gemfile.

Reply:
I found the issue, why the `gemfile_requirement` was not working correctly for the js gems. There was a small regexp issue. Resolved it and now working like a charm by using the default Gemfile.